package.json

```diff
@@ -2,11 +2,16 @@
   "name": "controlcenter",
   "version": "0.0.0",
   "dependencies": {
-    "react": "^0.12.2"
+    "es5-shim": "^4.1.0",
```

Comment:
"Like React, Reflux depends on an es5-shim for older browsers."

And yep it's not included as a dependency in reflux